### ISSUE TITLE
Compliance tracker calculations

### ISSUE THREAD

**Issue by MuhammadKhalilzadeh** [bluewave-labs/verifywise#394]
- In the compliance tracker page, the "Auditor completed" box will be removed.
- The calculations are as follows:
        - Total number of subcontrols: number of subcontrols in the compliance tracker. 
        - Implemented subcontrols: number of subcontrols where the user has entered at least one word in the following area
        - Compliance status: the percentage of two numbers (implemented subcontrols divided by total number of subcontrols%)

![Screenshot 2024-12-29 104747](https://github.com/user-attachments/assets/78b1287e-a47e-4af2-b202-668cb6fe8ec6)

**Comment by melpsh** [comment]
I don't understand how I should calculate these. :D The total number of subcontrols is fixed then, right?

**Comment by melpsh** [comment]
The total number of subcontrols is 104 now

**Comment by gorkem-bwl** [comment]
Yes, the total number of subcontrols is fixed, but that also comes from the backend.

**Comment by melpsh** [comment]
please provide the API that I can call to get this info

**Comment by HarshP4585** [comment]
The API endpoint for calculations is `<URL>/users/<user-id>/calculate-progress`

**Comment by melpsh** [comment]
Hi @HarshP4585, could you please provide an update on the status of this API? Have you had a chance to review the server logs and resolve the database issue?

**Comment by melpsh** [comment]
I've pulled the latest branch, but it seems that all the components related to the compliance tracker calculation have been deleted. Could you please clarify why this change was made?
![image](https://github.com/user-attachments/assets/dfec898f-e2b8-4483-9e90-5a04d83f0391)

**Comment by melpsh** [comment]
@MuhammadKhalilzadeh Thank you for checking, can you update me on the API as well?

**Comment by MuhammadKhalilzadeh** [comment]
> The API endpoint for calculations is `<URL>/users/<user-id>/calculate-progress`
> 
> CC: @melpsh

This provided already @melpsh
